tests/integration.rs

```diff
@@ -2908,7 +2908,8 @@ mod tests {
             handle_child_output(r, &output);
 
             let r = std::panic::catch_unwind(|| {
-                assert!(!String::from_utf8_lossy(&output.stdout).contains("cloud login:"));
+                assert!(!String::from_utf8_lossy(&output.stdout)
+                    .contains("Started OpenBSD Secure Shell server"));
```

Comment:
Can you make this a global `const &str` variable that can be reused across all tests?